packages/flame_audio/lib/bgm.dart

```diff
@@ -132,3 +132,15 @@ class Bgm extends WidgetsBindingObserver {
     }
   }
 }
+
+/// This allows a value of type T or T?
+/// to be treated as a value of type T?.
+///
+/// We use this so that APIs that have become
+/// non-nullable can still be used with `!` and `?`
+/// to support older versions of the API as well.
+///
+/// Todo(karvulf): Remove this after upgrading to flutter v3.0.0
```

Comment:
These should be removed or moved to a normal comment.

Reply:
I removed the line